src/figuresClasses.ts

```diff
@@ -2,10 +1,115 @@
+export interface Figure {
+  color: string;
+  a?: number;
+  b?: number;
+  c?: number;
+  radius?: number;
+  width?: number;
+  height?: number;
+}
+
+export type FiguresData = {
+  shape: string;
+  color: string;
+  area: number;
+};
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  public area = this.getArea();
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (
+      color === undefined ||
+      a === undefined ||
+      b === undefined ||
+      c === undefined
+    ) {
+      throw new Error('All values must be entered');
```

Comment:
[CODE STYLE] - When throwing an error, try to provide a more detailed description in the error message. For example, 'All constructor parameters for Triangle must be defined and non-null.'